mindsdb/api/http/namespaces/config.py

```diff
@@ -1,63 +1,106 @@
+import copy
+import traceback
+
 from flask import request
-from flask_restx import Resource
+from flask_restx import Resource, abort
```

Comment:
Any particular reason for using `abort` instead of just `return 400 'some err string'` ?